sdks/python/apache_beam/examples/snippets/snippets_test.py

```diff
@@ -31,7 +31,8 @@
 from apache_beam import typehints
 from apache_beam.transforms.util import assert_that
 from apache_beam.transforms.util import equal_to
-from apache_beam.utils.pipeline_options import TypeOptions
+# from apache_beam.utils.pipeline_options import TypeOptions
```

Comment:
Please take a pass and make sure to remove all commented out code left over from development/debugging.